hedera-node/hedera-app/src/test/resources/uploader/buckets-config.yaml

```diff
@@ -0,0 +1,4 @@
+blockStream:
+  buckets:
+    - { "name": "default-aws-bucket", "provider": "aws", "endpoint": "https://s3.amazonaws.com", "region": "us-east-1", "bucketName": "hedera-mainnet-blocks", "enabled": "true" }
+    - { "name": "default-gcp-bucket", "provider": "gcp", "endpoint": "https://storage.googleapis.com", "region": "", "bucketName": "hedera-mainnet-blocks", "enabled": "true" }
```

Comment:
We may not want to use `hedera-mainnet-blocks` as a name for a bucket in `test/resources` (here or other places, I've seen this name a couple of times)